packages/gatsby-source-wordpress/src/steps/check-plugin-requirements.ts

```diff
@@ -262,6 +263,7 @@ const prettyPermalinksAreEnabled = async ({
         }
       `,
       throwGqlErrors: true,
+      isFirstRequest: true,
```

Comment:
What does this flag do?

Reply:
It controls which error will be thrown. If we try to make GraphQL requests with isFirstRequest set the helper assumes an html or 404 response means WPGraphQL isn't active. Later on when making requests we can then differentiate and show a different error if a GraphQL request returns an html response - usually this means they have a plugin or custom code bug on the WP side.